jujugui/static/gui/src/app/components/boolean-config/boolean-config.js

```diff
@@ -92,8 +92,8 @@ YUI.add('boolean-config', function() {
               <div className="boolean-config--handle"></div>
             </label>
           </div>
-          <div className="boolean-config--description">
-            {this.props.option.description}
+          <div className="boolean-config--description"
+            dangerouslySetInnerHTML={{__html: this.props.option.description}}>
```

Comment:
The description is user supplied so we would need to do some kind of escaping for anything that is not an anchor. I just tried this with a local charm and it executes javascript, renders html etc.

Hopefully there is a lib or something that can help with this.